templates/dataset_binary.cpp.in

```diff
@@ -17,4 +17,12 @@ DataArray @NAME@(const DataArrayConstView &a, const DataArrayConstView &b) {
       intersection(a.attrs(), b.attrs()));
 }
 
+DataArray @NAME@(const DataArrayConstView &a, const VariableConstView &b) {
```

Comment:
What about the other order of arguments: `(Variable, Dataarray)`?
This could also be done in the bindings using the 'r' versions of the operators, e.g. `__radd__`.

Reply:
That is defined three lines below, not sure I understand the question.

Reply:
How did I miss this??